File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float cash;

    do
    {
        // Get change owed from user
        cash = get_float("Change owed: ");
    }
    while (cash < 0);

    // convert the dollar amount to cents
    int cents = round(cash * 100);
    int coins = 0;

    // check the amount of coins needed from the denomination 25, 10, 5, 1
    while (cents >= 25)
    {
        coins++;
        cents -= 25;
    }

    while (cents >= 10)
    {
        coins++;
        cents -= 10;
    }

    while (cents >= 5)
    {
        coins++;
        cents -= 5;
    }

    while (cents >= 1)
    {
        coins++;
        cents -= 1;
    }

    // show the amount of coins needed
    printf("%i", coins);
}

File: pset6/cash/cash.py
from cs50 import get_float

cash = get_float("Change owed: ")

while cash < 0:
    cash = get_float("Change owed: ")
else:
    #convert the dollar amount to cents
    cents = round(cash * 100);
    coins = 0;

    # check the amount of coins needed from the denomination 25, 10, 5, 1
    while cents >= 25:
        coins += 1
        cents -= 25

    while cents >= 10:
        coins += 1
        cents -= 10

    while cents >= 5:
        coins += 1
        cents -= 5

    while cents >= 1:
        coins += 1
        cents -= 1

    #show the amount of coins needed
    print(coins);